lib/elixir/lib/option_parser.ex

```diff
@@ -71,6 +71,9 @@ defmodule OptionParser do
   Note that you should only supply the `:switches` or the`:strict` option.
   If you supply both, an `ArgumentError` exception will be raised.
 
+    iex> OptionParser.parse(["--source", "lib"], switches: [source: :string], strict: [source: :string])
+    ** (ArgumentError) :switches and :strict cannot be given together
+
```

Comment:
well, a lot of the examples in the Elixir docs do exactly what this example does. put into code what it is written in words.